rest-api/tools/auth_setup.sh

```diff
@@ -86,6 +87,13 @@ function get_db_password {
   PASSWORD=`grep db_password $TMP_DB_INFO_FILE | cut -d\" -f4`
 }
 
+function get_geocode_key {
+  echo "Getting geocode api key ..."
+  tools/install_config.sh --key geocode_key --instance $INSTANCE \
```

Comment:
Do we not need to pass in --project and --account here?

We definitely want to pull this from cloud, and not from local datastore.

We also need to make sure that import_organizations.sh allows you to update the local database, while still geocoding using this key...

Reply:
We'll need to discuss this.